com.qq.QQ.yaml

```
@@ -0,0 +1,79 @@
+app-id: com.qq.QQ
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+command: qq
+separate-locales: false
+
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=x11'
+  - '--socket=pulseaudio'
+  - '--device=all'
+  - '--filesystem=xdg-documents:ro'
+  - '--filesystem=xdg-pictures:ro'
+  - '--filesystem=xdg-videos:ro'
+  - '--filesystem=xdg-download'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.kde.StatusNotifierWatcher'
+  - '--system-talk-name=org.freedesktop.login1'
+  - '--filesystem=xdg-run/pipewire-0'
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - '*.la'
+
+modules:
+  - name: qq
+    buildsystem: simple
+    build-commands:
+      - install apply_extra ${FLATPAK_DEST}/bin
+      - install -Dm755 qq.sh ${FLATPAK_DEST}/bin/qq
+      - install -Dm644 com.qq.QQ.metainfo.xml $FLATPAK_DEST/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm755 /usr/bin/ar -t ${FLATPAK_DEST}/bin
+      - install -Dm755 /usr/bin/desktop-file-edit -t ${FLATPAK_DEST}/bin
+      - install -Dm755 /usr/lib/$(gcc -print-multiarch)/libbfd-*.so -t ${FLATPAK_DEST}/lib
+    sources:
+      - type: script
+        commands:
+          - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
+          - exec zypak-wrapper /app/extra/QQ/qq "$@"
+        dest-filename: qq.sh
+
+      - type: script
+        commands:
+          - ar x qq.deb
+          - tar -xf data.tar.xz
+          - mv opt/QQ .
+          - mv usr export
+          - rm -rf export/share/icons
+          - install -Dm644 opt/QQ/resources/app/512x512.png export/share/icons/hicolor/512x512/apps/qq.png
```

```suggestion
          - install -Dm644 QQ/resources/app/512x512.png export/share/icons/hicolor/512x512/apps/qq.png
```
As `mv opt/QQ .`, remove `opt`.